src/components/procedure/_platform-add-repo.md

```diff
@@ -1,6 +1,14 @@
 Adding a repository from GitHub, GitLab, or BitBucket enables Semgrep Cloud Platform to perform many of its core features, such as the ability to record, triage, and manage findings.
 
-:::info Prerequisite
+* Azure Pipelines
+* Bitbucket
+* Buildkite
+* CircleCI
+* GitHub Actions
+* GitLab CI/CD
+* Jenkins
+
+:::info Prerequisites
 * **For GitHub or GitLab SaaS users:** A GitHub or GitLab SaaS repository associated with your account.
```

Comment:
I wonder if we should be putting these account + repository requirements in the quickstart, IF we are assuming that the user has gone through the quickstart BEFORE looking at Semgrep Code (and it would save us having redundancy in our docs, I think).

Reply:
I think it's a good idea to move this out, and maybe now's a good time to start a Prerequisites page that we can link to as necessary. It would also link to the supported languages doc, and it would minimize the amount of extra material on the quickstart/product TOFU docs. WDYT?

Reply:
IDK, is this too much? <img width="310" alt="Screenshot 2023-12-11 at 2 19 09 PM" src="https://github.com/semgrep/semgrep-docs/assets/16521651/156191af-b65f-473c-bfcb-4404b1869e35">

Reply:
What would be in the prereq page though? If it's just that callout it'd be pretty short:

>:::info Prerequisites
> * **For GitHub or GitLab SaaS users:** A GitHub or GitLab SaaS repository associated with your account.
> * **For BitBucket SaaS users:** A BitBucket repository and sufficient permissions to edit a BitBucket Pipeline and add repository variables.

Reply:
So I think the existing call out is a bit strange. I propose we have a prereqs page that includes this section

```

## CI/CD Systems

Semgrep supports integration with the following CI/CD systems to provide automated scanning of your repositories:

* Azure Pipelines
* Bitbucket
* Buildkite
* CircleCI
* GitHub Actions
* GitLab CI/CD
* Jenkins

When integrating Semgrep into your CI/CD system, ensure that your user account has sufficient permissions to modify pipelines and create and modify environment variables.
```

On the existing page, I have linked to the Add a SCM doc on the SCP page. As it stands, I think the callout is a bit vague and a bit too specific for the general nature of the section (it's the add a repo section, yes, but we talk about non-SCMs here too, like Jenkins)

Reply:
Resolving this, since we're working on prereqs as a separate task